src/loaders/workspace_uploader/workspace_uploader.py

```diff
@@ -843,22 +980,21 @@ def main():
         )
 
         start = time.time()
-        if create_assembly_only:
-            uploaded_count = _upload_assembly_files_in_parallel(
-                AssemblyUtil(
-                    conf.callback_url, service_ver=au_service_ver, token=conf.token
-                ),
-                ws,
-                env,
-                workspace_id,
-                load_id,
-                collections_source_dir,
-                wait_to_upload_objs,
-                batch_size,
-                source_dir,
-            )
-        else:
-            raise NotImplementedError("Genome object uploader is not implemented yet")
+
+        uploaded_count = _upload_objects_in_parallel(
+            ws,
+            env,
+            workspace_id,
+            load_id,
+            collections_source_dir,
+            wait_to_upload_objs,
+            batch_size,
+            source_dir,
+            conf,
```

Comment:
If it's just for simplicity in testing, I would leave it as is. I think we still should prioritize advancing quickly rather than spending time to local testing.

Reply:
It's already like that in the current code though. This change makes it harder to test

Reply:
👍

Reply:
:+1: